src/utils/filters.js

```diff
@@ -157,36 +144,46 @@ export function generateFilterOptions(layerData) {
   let datum;
 
   const layerFilter =
-    (layerData.layerObj &&
-      layerData.layerObj.aggregate &&
-      layerData.layerObj.aggregate.filter) ||
+    (layerData.layerObj && layerData.layerObj.aggregate && layerData.layerObj.aggregate.filter) ||
     (layerData.aggregate && layerData.aggregate.filter);
 
+  if (!layerFilter) {
+    return {};
+  }
```

Comment:
Is this the only code change? (The rest looks like just formatting changes?)

Reply:
Yes, the editor formatted on sav